src/consensus/fork_choice.zig

```diff
@@ -323,6 +336,10 @@ pub const ForkChoice = struct {
         return null;
     }
 
+    pub fn bestOverallSlot(self: *const ForkChoice) ?SlotAndHash {
```

Comment:
This is already implemented as https://github.com/Syndica/sig/blob/cbfbdc8f0ceef85928f246e72194f3108d8dc3e7/src/consensus/fork_choice.zig#L298

I remember during the review that added these methods, it was suggested to change the name. But maybe the original name should have been kept to avoid oversight like this...

Reply:
The renames did make it harder to figure out what was equivalent where, though that could also just be down to me not thoroughly reading the comments.

Maybe in cases like these we could use an alias for functions, ie here: `pub const bestOverallSlot = heaviestOverallSlot;`, or `pub const bestOverallSlot = @compileError("Use heaviestOverallSlot instead");`

Reply:
523c46d2167767976942308aff0823ed8575668c